src/ristretto/ristretto_keys.rs

```diff
@@ -219,6 +219,13 @@ impl RistrettoPublicKey {
             compressed: pk.compress(),
         }
     }
+
+    pub(crate) fn new_from_compressed(compressed: CompressedRistretto) -> Option<RistrettoPublicKey> {
```

Comment:
Feel like this should return a `Result` since our contract is that we always store a valid compressed/decompressed pair.

Reply:
Cool, agree

Reply:
`map(..).ok_or(E)` will do it

Reply:
There isn't an Error struct yet in the crate for this kind of error, and the method is only called privately from Deserialize which already exposes a ByteArrayError, so I restricted it further to `super` to reflect that it is private only and kept it as the `Option` that dalek exposes.